packages/patrol_finders/test/patrol_tester_test.dart

```diff
@@ -211,6 +292,87 @@ void main() {
           throwsAssertionError,
         );
       });
+
+      group('with alignment', () {
+        patrolWidgetTest('finds no widgets', (tester) async {
+          const width = 300.0;
+          setPhysicalSize(tester.tester, width);
+
+          await tester.pumpWidget(
+            MaterialApp(
+              home: Scaffold(
+                body: SingleChildScrollView(
+                  scrollDirection: Axis.horizontal,
+                  child: Container(
+                    color: Colors.blue,
+                    width: 2 * width,
+                    child: const Row(
+                      mainAxisAlignment: MainAxisAlignment.center,
+                      children: [
+                        SizedBox(
+                          width: width,
+                          child: TextField(),
+                        ),
+                      ],
+                    ),
+                  ),
+                ),
+              ),
+            ),
+          );
+
+          await expectLater(
+            () => tester.enterText(
+              find.byType(TextField),
+              'text',
+              alignment: Alignment.centerRight,
+            ),
+            throwsA(isA<WaitUntilVisibleTimeoutException>()),
+          );
+        });
+
+        patrolWidgetTest('finds widgets', (tester) async {
+          const width = 300.0;
+          setPhysicalSize(tester.tester, width);
+
+          final controller = TextEditingController();
+
+          await tester.pumpWidget(
+            MaterialApp(
+              home: Scaffold(
+                body: SingleChildScrollView(
+                  scrollDirection: Axis.horizontal,
+                  child: Container(
+                    color: Colors.blue,
```

Comment:
Is color necessary?

Reply:
No, was needed for debugging, will remove